packages/shared/src/hooks/squads/useSquad.ts

```diff
@@ -19,8 +19,8 @@ interface UseSquad {
 }
 
 export const useSquad = ({ handle }: UseSquadProps): UseSquad => {
-  const { isFetched: isBootFetched } = useContext(AuthContext);
-  const queryKey = ['squad', handle];
+  const { isFetched: isBootFetched, user } = useContext(AuthContext);
+  const queryKey = ['squad', handle, user?.id];
```

Comment:
invalidateQueries with `['squad', handle]` still work because we use array query keys.